modules/statistics/php/NDB_Menu_statistics_site.class.inc

```diff
@@ -38,7 +38,8 @@ class NDB_Menu_Statistics_Site extends NDB_Menu
      */
     function _hasAccess($centerID)
     {
-        $user =& User::singleton();
+        $user     =& User::singleton();
+        $centerID =$user->getData('CenterID');
```

Comment:
use `user->getCenterIDs`

the code you added does not take into account multi-sites either